modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -1369,7 +1378,7 @@ GrainTracker::updateFieldInfo()
       {
         const Elem * elem = mesh.elem(entity);
         std::vector<Point> centroid(1, elem->centroid());
-        if (_ebsd_reader)
+        if (_ebsd_reader && _first_time)
```

Comment:
I've looked at this problem quite a bit and I don't think it's easily solvable without a lot more work. I suggest for now that we move on and come back to this one. I'll open a ticket to deal with splitting grains. @frombs - I don't think this will prevent you from running your simulations.